pkg/controllers/apps/base/base.go

```diff
@@ -193,6 +194,30 @@ func (c *Controller) deleteDescription(obj interface{}) {
 	c.enqueue(base)
 }
 
+func (c *Controller) updateDescription(old, cur interface{}) {
```

Comment:
No need to update status of base.

Reply:
> No need to update status of base.

As we discussed, I will refactor this implementation.